doc/rados/operations/erasure-code.rst

```diff
@@ -67,6 +67,16 @@ profile can be displayed with this command:
    crush-failure-domain=host
    technique=reed_sol_van
 
+.. note::
+   The default erasure-coded pool, the profile of which is displayed here, is
+   not the same as the simplest erasure-coded pool. 
+   
+   The default erasure-coded pool has two data chunks (k) and two coding chunks
+   (m). The profile of the default erasure-coded pool is "k=2 m=2".
+
+   The simplest erasure-coded pool has two data chunks (k) and one coding chunk
+   (m). The profile of the simplest erasure-coded pool is "k=2 m=1".
```

Comment:
Do we want to add a note about how m=1 is a bad idea?

Reply:
@anthonyeleven Definitely, when we organize this page and do rewrites.  Note to future me: remember to define k and m as data chunks and coding chunks before k and m are mentioned in the text.